File: inui/foundation/blocks/circle-graph/circle-graph.js
$("[data-circle-graph]").each(function() {
  var $graph = $(this),
      percent = parseInt($graph.data('percent'), 10),
      deg = 360*percent/100;
  if(percent > 50) {
    $graph.addClass('gt-50');
  }
  $graph.find('.circle-graph-progress-fill').css('transform','rotate('+ deg +'deg)');
  $graph.find('.circle-graph-percents-number').html(percent+'%');
});

File: inui/foundation/blocks/circle_graph/circle_graph.py
from inui.elements import *
from inui.svg import *
Html(

    data = (
        Body(

            data = (
                Div(
                    classs='''circle-graph''',
                    data_circle_graph='',
                    data_percent='''50''',
                    data = (
                        Div(
                            classs='''circle-graph-progress''',
                            data = (
                                Div(
                                    classs='''circle-graph-progress-fill''',
                                ), )
                        ), 
                        Div(
                            classs='''circle-graph-percents''',
                            data = (
                                Div(
                                    classs='''circle-graph-percents-wrapper''',
                                    data = (
                                        Span(
                                            classs='''circle-graph-percents-number''',
                                        ), 
                                        Span(
                                            classs='''circle-graph-percents-units''',
                                            data = ('''of 100''',)
                                        ), )
                                ), )
                        ), )
                ), )
        ), )
)